src/style-installer.js

```diff
@@ -116,29 +116,38 @@ export function makeBodyVisible(doc, opt_waitForServices) {
       animation: 'none',
     });
   };
-  /** @const {!Window} */
-  const win = doc.defaultView;
-  documentStateFor(win).onBodyAvailable(() => {
-    if (win[bodyVisibleSentinel]) {
-      return;
-    }
-    win[bodyVisibleSentinel] = true;
-    if (opt_waitForServices) {
-      waitForServices(win).catch(() => []).then(services => {
+  try {
+    /** @const {!Window} */
+    const win = doc.defaultView;
+    documentStateFor(win).onBodyAvailable(() => {
+      if (win[bodyVisibleSentinel]) {
+        return;
+      }
+      win[bodyVisibleSentinel] = true;
+      if (opt_waitForServices) {
+        waitForServices(win).catch(() => []).then(services => {
```

Comment:
Isn't this `catch` here potentially swallowing exceptions? should we also throwAsync in here?